File: SiHyeon/week12/JadenCase 문자열 만들기.cpp
#include <string>
#include <vector>

using namespace std;

string solution(string s)
{
    string answer = "";
    answer += toupper(s[0]); // 첫글자 대문자
    for (int i = 1; i < s.size(); i++)
    {
        // char c = tolower(s[i]);
        // if (s[i-1] == ' ')
        //     c = toupper(s[i]);
        // answer += c;
        answer += s[i - 1] == ' ' ? toupper(s[i]) : tolower(s[i]);
    }
    return answer;
}

File: seoyeon/week12/jadencase 문자열 만들기.py
def solution(s):
    answer = ''
    tmp = s.lower()
    idx = 0
    for i in tmp :
        if idx == 0 :
            answer += i.upper()
        else :
            answer += i
        idx += 1
        if i == ' ' :
            idx = 0
    return answer